.circleci/config.yml

```diff
@@ -770,33 +805,43 @@ workflows:
             - Functional Tests - Playwright (PR)
             - Deploy Storybooks (PR)
 
-  production_smoke_test:
-    # This workflow can be useful if we want to run test changes in our functional tests
-    # against deployed code. It can be triggered by setting a pipeline parameter to true.
-    when: << pipeline.parameters.run-smoke-tests >>
+  # Triggered remotely. See .circleci/README.md
+  production_smoke_tests:
+    when: << pipeline.parameters.enable_production_smoke_tests >>
     jobs:
-      - production-smoke-tests
-      - test-content-server-remote
+      # Note that these target stage too! This might seem odd, but having tests pass on staging is currently
+      # a requirement for all production deploys.
+      - test-content-server-remote-part-0:
+          name: Smoke Stage - Content Server Part 1
+      - test-content-server-remote-part-1:
+          name: Smoke Stage - Content Server Part 2
+      - test-content-server-remote-part-2:
+          name: Smoke Stage - Content Server Part 2
+      - smoke-tests:
+          name: Smoke Test Stage - Playwright
+          target: test-stage
+      - smoke-tests:
```

Comment:
q: Should these have different names (L820 vs L823)?
ie: `smoke-tests-stage` vs `smoke-tests-prod`